src/xbot/common/math/PIDManagerFactory.java

```diff
@@ -0,0 +1,45 @@
+package xbot.common.math;
+
+import com.google.inject.assistedinject.Assisted;
+
+public interface PIDManagerFactory {
```

Comment:
I can see how writing the factory here plus the constructors elsewhere is kind of a pain, but man, what a time savings elsewhere.